function/services/eligibility-checker/index.js

```diff
@@ -4,6 +4,38 @@ const {DateTime} = require('luxon');
 const invalidInjuryCodes = require('../../constants/ineligible-injury-codes');
 const {setPreviouslyAppliedEligibility} = require('../question-helpers/previous-application');
 
+function checkEligibilityForInvalidInjuries(applicationFormJson) {
+    let ineligibleDueToInjuries = false;
+    const applicationType = applicationFormJson?.application_type;
+    if (
+        applicationFormJson?.injury_details_code &&
+        (applicationType === 2 || applicationType === 3)
+    ) {
+        ineligibleDueToInjuries = true;
+        const injuryCodes = applicationFormJson.injury_details_code.split(':');
+        injuryCodes.forEach(code => {
+            if (!invalidInjuryCodes.includes(code)) {
+                ineligibleDueToInjuries = false;
+            }
+        });
+    }
+    return ineligibleDueToInjuries;
+}
+
+function checkEligibilityForNoInjuries(applicationFormJson) {
+    let noInjuries = false;
+    const applicationType = applicationFormJson?.application_type;
+    if (applicationType === 2 || applicationType === 3) {
```

Comment:
Replace with meaningfull constants for applicationType

Reply:
Happy to do this, do you know where the different application types are stored so I can get them right?

Reply:
Sorry just saw this, no I don't but I will have a look